### ISSUE TITLE
joblib verbosity

### ISSUE THREAD

**Issue by choltz95** [rtavenar/tslearn#139]
Thanks for 0.2! When I run a cdist on a large data-set, I get anxious not knowing the progress. Joblib has a verbosity setting that prints # jobs completed & time elapsed depending verbosity setting. It might be useful to include this parameter for the parallel cdist and pass it straight to the calls to joblib Parallel().

**Comment by johannfaouzi** [comment]
It seems like a good idea to add a `verbose` parameter every time there is a `n_jobs` parameter:
* [tslearn.metrics.cdist_dtw](https://tslearn.readthedocs.io/en/latest/gen_modules/metrics/tslearn.metrics.cdist_dtw.html#tslearn.metrics.cdist_dtw)
* [tslearn.metrics.cdist_gak](https://tslearn.readthedocs.io/en/latest/gen_modules/metrics/tslearn.metrics.cdist_gak.html#tslearn.metrics.cdist_gak)

I think that this is the case for all the estimators in scikit-learn (all the ones that I have in mind at least).

**Comment by rtavenar** [comment]
Hi @choltz95 

Nice suggestion indeed. I was going to ask how it is dealt with in sklearn, but @johannfaouzi was faster than me.

So, I think this is a good idea. I will not be able to invest time on that in the near future, though. Do you think you could open a PR on that topic @choltz95 ? If so, I could probably review it.

**Comment by choltz95** [comment]
Sure no problem.

**Comment by rtavenar** [comment]
@choltz95 if you want to work on that PR, do not hesitate to ask for help. Also, I suggest you branch from dev because this is the branch that is used to prepare v0.3 in which your suggestion could be included.

**Comment by choltz95** [comment]
Sorry for the delay. Opened a pull request for review & waiting for tests to finish.

**Comment by rtavenar** [comment]
This is now merged to `dev` branch (will be available in tslearn v0.3 on PyPI), thanks to the amazing work by @choltz95 !

**Comment by zaemyung** [comment]
Is the verbosity feature working? Even when I set `verbose` to 1, I don't see any progress bar.
```python
km = TimeSeriesKMeans(n_clusters=num_clusters, metric=metric, max_iter_barycenter=3, verbose=1, n_jobs=-1, tol=tolerance, max_iter=3, random_state=random_state).fit(X)
